diagnostic.md

```diff
@@ -184,5 +227,10 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
+i believe it would be 25% because the min width for 25 is 400px and the min width for 30% is 800px
```

Comment:
At a screen width of 550px, the `@media(min-width= 400px)` would take over and `.stretch` is 25%. Thus 25% * 550px = 137.50px.